tortoise/models.py

```diff
@@ -783,3 +813,28 @@ class Meta:
                     table="custom_table"
                     unique_together=(("field_a", "field_b"), )
         """
+
```

Comment:
The `Meta` nested class here is only for documentation. At the moment all the processed fields end up in the `_meta` (which is a `MetaInfo` class)
:thinking: Possibly we move the documentation from the `.rst` file into here, and also specify default values, then we don't have to deal with `.get()` so much... :thinking:

Reply:
I found this `Meta` class is useful to store the defaults and  their descriptions here, but if you have better or standard place for them, please help in it. 

These  settings are different from the _meta ones, because they are independent from the Tortoise model, and should not be in the `_meta` at all imo. It could be better to separate them into a different class? e.g `class PydanticMeta:`?

Reply:
Unsure, As long as there is adequate namespacing, it should be fine.
This was me thinking of moving the field docs into `Meta` (and then also provide defaults there instead of in code via e.g. `.get("indexes", [])`).

A PydanticMeta might make more sense? Then there is a clearer separation of concerns?